### ISSUE TITLE
Not Working!

### ISSUE THREAD

**Issue by vivekvinodh** [mootcycle/slackporter#2]
error: emojilist all handler Error:
undefined

I'm getting this error when I'm trying to run the node server. 
Any ideas?

**Comment by kjhobin** [comment]
I'm also getting this same error, a few minutes after trying to get the Emoji.

**Comment by TApicella** [comment]
Same here. I have made emoji for all 150 original pokemon on another channel, I really want to be able to transfer them haha

**Comment by forgottentea-xx** [comment]
same

**Comment by Willian-Zhang** [comment]
same here

**Comment by TApicella** [comment]
I hope this isn't rude of me to post this, but I wrote up two hacky ways to bulk download and upload emoji from Slack. This is a VERY crude method using Selenium, but it works. 

Download: https://gist.github.com/TApicella/564a917e6815f39e7ddc
Upload: https://gist.github.com/TApicella/f68d67796528f286adf3

**Comment by mikesigs** [comment]
I was getting the same error. Yours could literally be from almost anything, but I found mine had to do with not having a valid API token. It looks like the app tries to request one, but it doesn't work. I'm still digging into whether this is possible or not. But a quick solution is to go to https://api.slack.com and generate test tokens for the Slacks you're targetting.

**Comment by Marek-Bernad** [comment]
Nothing happens after clicking "Get Some Emoji". Just I clicked on that button and nothing happen, credentials are verified and well typed. This was tested on firefox and chrome, nothing work. Could be some problem if I am creator of both teams and want to send emoji from team to team with same slack Account email?

**Comment by marcthayer** [comment]
Can someone provide a little more setup instruction than what exists in the readme? Also, if you're still having issues, try checking out @radicand's potential fixes in https://github.com/mootcycle/slackporter/pull/3.

**Comment by peabnuts123** [comment]
As @mikesigs said; if you add Tokens yourself, the script runs better. Go to [this page](https://api.slack.com/custom-integrations/legacy-tokens) to add tokens to your Slacks (it will ask you to re-type your password). After this page displays tokens on it instead of "None" this works better.

I still had some issues after this getting it to actually work, but I found running the code in debug mode in vscode actually got it to work for some reason. YMMV.

**Edit:** Oh, I also used [radicand's updated branch](https://github.com/radicand/slackporter/tree/fix/2) to run this.
